src/main/resources/templates/recipe/create.html

```diff
@@ -54,13 +54,14 @@
 				<label th:text="${category.name}"></label>
 			</th:block>
 		</div>
+
 		<div class="form-group">
-			<label th:for="beverages">Sommelier Suggestions</label>
-			<th:block th:each="beverage : ${beverages}">
-				<input type="checkbox" name="beverages" th:value="${beverage.id}">
-				<label th:text="${beverage.name}"></label>
-			</th:block>
+			<label>Created By
```

Comment:
We could probably remove this part of the create form! My thinking is that all of the user ownership stuff should happen behind the scenes. There's no need to have a box available to see who is creating a recipe because when you log in that is who the recipe should be tied to.